.github/ISSUE_TEMPLATE/1-feature-request.yml

```diff
@@ -0,0 +1,66 @@
+name: Feature request
+description: Submit a new idea
+title: "[Idea]: "
```

Comment:
[nit] the other two forms use "Bug: " and "Docs: " rather than "[Bug]: " or "[Docs]: "